### ISSUE TITLE
PatchModelAddDownscale (Kohya Deep Shrink) - broke after last update (Feb 24)

### ISSUE THREAD

**Issue by DavidEBell** [comfyanonymous/ComfyUI#2903]
I ran my daily update first thing Feb 24 and my workflow stopped working.

Macintosh OSX

Error Message:
Error occurred when executing KSampler: The operator 'aten::upsample_bicubic2d.out' is not currently implemented for the MPS device. If you want this op to be added in priority during the prototype phase of this feature, please comment on https://github.com/pytorch/pytorch/issues/77764. As a temporary fix, you can set the environment variable PYTORCH_ENABLE_MPS_FALLBACK=1 to use the CPU as a fallback for this op. WARNING: this will be slower than running natively on MPS. File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/execution.py", line 152, in recursive_execute output_data, output_ui = get_output_data(obj, input_data_all) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/execution.py", line 82, in get_output_data return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/execution.py", line 75, in map_node_over_list results.append(getattr(obj, func)(**slice_dict(input_data_all, i))) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/nodes.py", line 1368, in sample return common_ksampler(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, denoise=denoise) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/nodes.py", line 1338, in common_ksampler samples = comfy.sample.sample(model, noise, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/custom_nodes/ComfyUI-AnimateDiff-Evolved/animatediff/sampling.py", line 248, in motion_sample return orig_comfy_sample(model, noise, *args, **kwargs) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/custom_nodes/ComfyUI-Impact-Pack/modules/impact/sample_error_enhancer.py", line 22, in informative_sample raise e File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/custom_nodes/ComfyUI-Impact-Pack/modules/impact/sample_error_enhancer.py", line 9, in informative_sample return original_sample(*args, **kwargs) # This code helps interpret error messages that occur within exceptions but does not have any impact on other operations. File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/sample.py", line 100, in sample samples = sampler.sample(noise, positive_copy, negative_copy, cfg=cfg, latent_image=latent_image, start_step=start_step, last_step=last_step, force_full_denoise=force_full_denoise, denoise_mask=noise_mask, sigmas=sigmas, callback=callback, disable_pbar=disable_pbar, seed=seed) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/samplers.py", line 706, in sample return sample(self.model, noise, positive, negative, cfg, self.device, sampler, sigmas, self.model_options, latent_image=latent_image, denoise_mask=denoise_mask, callback=callback, disable_pbar=disable_pbar, seed=seed) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/samplers.py", line 611, in sample samples = sampler.sample(model_wrap, sigmas, extra_args, callback, noise, latent_image, denoise_mask, disable_pbar) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/samplers.py", line 550, in sample samples = self.sampler_function(model_k, noise, sigmas, extra_args=extra_args, callback=k_callback, disable=disable_pbar, **self.extra_options) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context return func(*args, **kwargs) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/k_diffusion/sampling.py", line 580, in sample_dpmpp_2m denoised = model(x, sigmas[i] * s_in, **extra_args) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl return self._call_impl(*args, **kwargs) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl return forward_call(*args, **kwargs) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/samplers.py", line 282, in forward out = self.inner_model(x, sigma, cond=cond, uncond=uncond, cond_scale=cond_scale, model_options=model_options, seed=seed) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl return self.call_impl(*args, **kwargs) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1520, in call_impl return forward_call(*args, **kwargs) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/samplers.py", line 272, in forward return self.apply_model(*args, **kwargs) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/samplers.py", line 269, in apply_model out = sampling_function(self.inner_model, x, timestep, uncond, cond, cond_scale, model_options=model_options, seed=seed) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/samplers.py", line 249, in sampling_function cond_pred, uncond_pred = calc_cond_uncond_batch(model, cond, uncond, x, timestep, model_options) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/samplers.py", line 223, in calc_cond_uncond_batch output = model.apply_model(input_x, timestep, **c).chunk(batch_chunks) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/model_base.py", line 91, in apply_model model_output = self.diffusion_model(xc, t, context=context, control=control, transformer_options=transformer_options, **extra_conds).float() File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl return self._call_impl(*args, **kwargs) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl return forward_call(*args, **kwargs) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/ldm/modules/diffusionmodules/openaimodel.py", line 858, in forward h = p(h, transformer_options) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy_extras/nodes_model_downscale.py", line 30, in input_block_patch h = comfy.utils.common_upscale(h, round(h.shape[-1] * (1.0 / downscale_factor)), round(h.shape[-2] * (1.0 / downscale_factor)), downscale_method, "disabled") File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/comfy/utils.py", line 404, in common_upscale return torch.nn.functional.interpolate(s, size=(height, width), mode=upscale_method) File "/Volumes/Working_02d/600_AI_work/Pinockio_Actual/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/nn/functional.py", line 4073, in interpolate return torch._C._nn.upsample_bicubic2d(input, output_size, align_corners, scale_factors)
